clio-server/src/test/scala/org/broadinstitute/clio/server/service/MockIndexService.scala

```diff
@@ -32,18 +32,25 @@ abstract class MockIndexService[
       elasticsearchIndex,
       clioIndex
     ) {
-  val queryCalls = ArrayBuffer.empty[clioIndex.QueryInputType]
-  val queryAllCalls = ArrayBuffer.empty[clioIndex.QueryInputType]
-  val upsertCalls = ArrayBuffer.empty[(clioIndex.KeyType, clioIndex.MetadataType)]
+
+  val queryCalls: ArrayBuffer[clioIndex.QueryInputType] =
+    ArrayBuffer.empty[clioIndex.QueryInputType]
```

Comment:
Is this just to make intellij be quiet? Have considered doing it myself on passes through here, but the type annotation is so closely mirrored by the invocation that it seems kind of verbose